runtime/php74/cpu-arm.Dockerfile

```diff
@@ -0,0 +1,230 @@
+# RUN cp /lib64/libunistring.so.0 /bref/lib/libunistring.so.0
+# RUN cp /lib64/libsasl2.so.3 /bref/lib/libsasl2.so.3
+# RUN cp /lib64/libssl3.so /bref/lib/libssl3.so
+# RUN cp /lib64/libsmime3.so /bref/lib/libsmime3.so
+
+# PHP Binary
+RUN cp /usr/bin/php /bref/bin/php && chmod +x /bref/bin/php
+RUN cp /lib64/libedit.so.0 /bref/lib/libedit.so.0
+RUN cp /lib64/libncurses.so.6 /bref/lib/libncurses.so.6
+#RUN cp /lib64/libcrypt.so.1 /bref/lib/libcrypt.so.1
+#RUN cp /lib64/libresolv.so.2 /bref/lib/libresolv.so.2
+#RUN cp /lib64/libm.so.6 /bref/lib/libm.so.6
+#RUN cp /lib64/libdl.so.2 /bref/lib/libdl.so.2
+#RUN cp /lib64/libgssapi_krb5.so.2 /bref/lib/libgssapi_krb5.so.2
+#RUN cp /lib64/libkrb5.so.3 /bref/lib/libkrb5.so.3
+#RUN cp /lib64/libk5crypto.so.3 /bref/lib/libk5crypto.so.3
+#RUN cp /lib64/libcom_err.so.2 /bref/lib/libcom_err.so.2
+#RUN cp /lib64/libcrypto.so.10 /bref/lib/libcrypto.so.10
+#RUN cp /lib64/libc.so.6 /bref/lib/libc.so.6
+#RUN cp /lib64/libpthread.so.0 /bref/lib/libpthread.so.0
+#RUN cp /lib64/ld-linux-x86-64.so.2 /bref/lib/ld-linux-x86-64.so.2
+#RUN cp /lib64/libgcc_s.so.1 /bref/lib/libgcc_s.so.1
+#RUN cp /lib64/liblzma.so.5 /bref/lib/liblzma.so.5
+#RUN cp /lib64/libkrb5support.so.0 /bref/lib/libkrb5support.so.0
+#RUN cp /lib64/libkeyutils.so.1 /bref/lib/libkeyutils.so.1
+#RUN cp /lib64/libtinfo.so.6 /bref/lib/libtinfo.so.6
+#RUN cp /lib64/libpcre.so.1 /bref/lib/libpcre.so.1
+
+# Default Extensions
+RUN cp /usr/lib64/php/modules/json.so /bref/php-modules/json.so
+RUN cp /usr/lib64/php/modules/ctype.so /bref/php-modules/ctype.so
+RUN cp /usr/lib64/php/modules/exif.so /bref/php-modules/exif.so
+RUN cp /usr/lib64/php/modules/fileinfo.so /bref/php-modules/fileinfo.so
+RUN cp /usr/lib64/php/modules/ftp.so /bref/php-modules/ftp.so
+RUN cp /usr/lib64/php/modules/gettext.so /bref/php-modules/gettext.so
+RUN cp /usr/lib64/php/modules/iconv.so /bref/php-modules/iconv.so
+RUN cp /usr/lib64/php/modules/sockets.so /bref/php-modules/sockets.so
+RUN cp /usr/lib64/php/modules/tokenizer.so /bref/php-modules/tokenizer.so
+
+# cURL
+RUN cp /usr/lib64/php/modules/curl.so /bref/php-modules/curl.so
+#RUN cp /lib64/libcurl.so.4 /bref/lib/libcurl.so.4
+#RUN cp /lib64/libnghttp2.so.14 /bref/lib/libnghttp2.so.14
+#RUN cp /lib64/libidn2.so.0 /bref/lib/libidn2.so.0
+#RUN cp /lib64/libldap-2.4.so.2 /bref/lib/libldap-2.4.so.2
+#RUN cp /lib64/liblber-2.4.so.2 /bref/lib/liblber-2.4.so.2
+#RUN cp /lib64/libnss3.so /bref/lib/libnss3.so
+#RUN cp /lib64/libnssutil3.so /bref/lib/libnssutil3.so
+#RUN cp /lib64/libplds4.so /bref/lib/libplds4.so
+#RUN cp /lib64/libplc4.so /bref/lib/libplc4.so
+#RUN cp /lib64/libnspr4.so /bref/lib/libnspr4.so
+
+# sodium
+RUN cp /usr/lib64/php/modules/sodium.so /bref/php-modules/sodium.so
+RUN cp /usr/lib64/libsodium.so.23 /bref/lib/libsodium.so.23
+
+COPY runtime/tests/test_1_binary.php /bref/tests/
+COPY runtime/tests/test_2_default_extensions.php /bref/tests/
+
+FROM binary as extensions
```

Comment:
I tried to explain the Docker file structure (along with the phpxx folder) here: https://github.com/brefphp/bref/tree/runtime/runtime#the-phpxx-folder

In summary, it helps a lot to break things down into multiple docker stage so that we can test things separately. An automation test can target just the `binary` image to check if the PHP binary is working. A later test can target the `extensions` stage. When tests are not passing, it's much easier to find what's wrong if they are separate rather than trying to find what's wrong with an entire image that has been fully built already. Even if the extensions are broken, the `binary` stage can still be built and tested successfully.